examples/specs/normalized/layer_box_plot_circle_normalized.vl.json

```diff
@@ -1,155 +0,0 @@
-{
```

Comment:
I think you forgot to re-compile the normalize script?

Reply:
I used npm run build:example... is that not enough?